src/waypoint.jsx

```diff
@@ -32,6 +45,20 @@ function debugLog() {
   console.log(arguments); // eslint-disable-line no-console
 }
 
+function parseOffsetAsPixels(str) {
```

Comment:
> I don't think computing these over and over affect performance too much, but it's an easy change.

As an aside, I don't think either approach causes anything to be computed or created more than the other.

Most JS module loaders cache the modules, so these will be created the first time that the lib is required and never again. And if we move it into the class, they'll be defined on the prototype, also just once. So it should just be a matter of preference, which could be an aesthetic preference or one based on whether it makes sense to expose these to consumers of the lib.